specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -992,6 +1052,21 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-access-soft-deleted-resources",
```

Comment:
With this parameter , does it mean only soft deleted items are returned, or all objects include soft deleted ones are returned?
Please update description to include this.

Reply:
This header is just to get soft deleted resources, volume or volume group. If this header is used with active volume or volume group, it will return 404.

Reply:
Not applicable now